000-debug/logger.php

```diff
@@ -43,26 +43,31 @@ function l( $stuff = null ) {
 	static $pageload;
 	// Call l() on each argument
 	if ( func_num_args() > 1 ) {
-		foreach ( func_get_args() as $arg )
-			l($arg);
+		foreach ( func_get_args() as $arg ) {
+			l( $arg );
+		}
 		return $stuff;
 	}
-	if ( !isset( $pageload ) ) {
+	if ( ! isset( $pageload ) ) {
 		$pageload = substr( md5( mt_rand() ), 0, 4 );
-		if ( !empty( $_SERVER['argv'] ) )
+		if ( ! empty( $_SERVER['argv'] ) ) {
 			$hint = implode( ' ', $_SERVER['argv'] );
-		elseif ( isset( $_SERVER['HTTP_HOST'] ) && isset( $_SERVER['REQUEST_URI'] ) )
+		} elseif ( isset( $_SERVER['HTTP_HOST'] ) && isset( $_SERVER['REQUEST_URI'] ) ) {
 			$hint = $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
-		else
+		} else {
 			$hint = php_sapi_name();
-		error_log( sprintf( "[%s-%s => %s]", $pageload, getmypid(), $hint ) );
+		}
+		// phpcs:disable WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
Seemed like logger needs to do `error_log` so I ignored the warning here, but feel free to correct me on that.